bindata/cert-manager-deployment/cert-manager-controller/cert-manager-deployment.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+    labels:
+        app: cert-manager
+        app.kubernetes.io/component: controller
+        app.kubernetes.io/instance: cert-manager
+        app.kubernetes.io/name: cert-manager
+    name: cert-manager
+    namespace: cert-manager
```

Comment:
Shouldn't we be using openshift-cert-manager? Or would we need special privileges to create such a NS?

Reply:
We agreed using other channels to leave it as is.